Trove.js

```diff
@@ -0,0 +1,326 @@
+{
+	"translatorID": "8efcb7cb-4180-4555-969a-08e8b34066c4",
+	"label": "Trove",
+	"creator": "Tim Sherratt",
+	"target": "^https?://trove\\.nla\\.gov\\.au/(?:newspaper|gazette|work|book|article|picture|music|map|collection)/",
+	"minVersion": "2.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsbv",
+	"lastUpdated": "2016-04-04 09:40:10"
+}
+
+/*
+   Trove Translator
+   Copyright (C) 2016 Tim Sherratt (tim@discontents.com.au, @wragge)
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Affero General Public License for more details.
+
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(doc, url) {
+	if (url.match(/\/result\?/) || url.match(/\/newspaper\/page/)) {
```

Comment:
here and below, we prefer using .indexOf("string") != -1 to test for literal strings and .search(/regex/) != -1 where you need rege capabilities. It's a lot more efficient.